### PR TITLE
Simplify mnist example.

### PR DESCRIPTION
**Pull request by youxingling**
### What changes were proposed in this pull request?

Simplify mnist example.

### Why are the changes needed?

Help users run more easily.

### Does this PR introduce any user-facing change?

No.

### How was this patch tested?

UT.

### CODE REVIEW

File: scripts/build_wheel.sh [hunk 1]
```diff
@@ -21,5 +21,5 @@ make -f dlrover/Makefile
 
 # Create dlrover package
 echo "Building the wheel for dlrover."
-rm -rf ./build/lib
```
Comment (workingloong) [inline scripts/build_wheel.sh:24]: Why do we need to remove the line?

File: examples/pytorch/mnist/cnn_train.py [hunk 2]
```diff
@@ -98,21 +97,20 @@ def train(args):
     """
     setup()
 
-    train_data = torchvision.datasets.ImageFolder(
-        root=args.training_data,
-        transform=transforms.ToTensor(),
+    train_dataset = datasets.MNIST(
```
Comment (workingloong) [inline examples/pytorch/mnist/cnn_train.py:100]: It is better to use a flag to download the MNIST dateset. Sometimes, it is very slow to download the MNIST dataset. For example:

```Python
if args.training_data:
    train_dataset = torchvision.datasets.ImageFolder(
          root=args.training_data,
          transform=transforms.ToTensor(),
else:
    train_dataset = datasets.MNIST(
        "./data", train=True, transform=transforms.ToTensor(), download=True
    )
```

File: examples/pytorch/mnist/README.md [hunk 3]
```diff
@@ -3,20 +3,6 @@
 The document describes how to use DLRover to train a Pytorch CNN model
 with MNIST dataset.
 
-## Prepare Data
```
Comment (workingloong) [inline examples/pytorch/mnist/README.md:6]: It is better to retain the introduction and add a way to create the dataset by `datasets.MNIST()`.

File: scripts/build_wheel.sh (hunk 1 above)
Comment (youxingling) [reply scripts/build_wheel.sh:24]: Just to reduce misunderstandings. Before this line, we have already executed `rm -rf build`. And the build file doesn't exist until we execute `python setup.py`.

Comment (workingloong) [reply scripts/build_wheel.sh:24]: The command will not raise an error even if the `./build/lib` does not exist. So, it is better to retain it to clear the cache.

File: examples/pytorch/mnist/README.md [hunk 4]
```diff
@@ -27,6 +29,13 @@ pip install dlrover -U
 
 Then, we can use `dlrover-run` to start the training by
 
+```bash
+dlrover-run --standalone --nproc_per_node=${GPU_NUM} \
```
Comment (workingloong) [inline examples/pytorch/mnist/README.md:33]: Now, we don't need to set `--standalone` to run locally. 
`dlrover-run --nproc_per_node=${GPU_NUM}` is OK.

**Review by workingloong — APPROVED** [review]
LGTM

**Pull request merged**